doc/source/authentication.rst

```diff
@@ -83,16 +83,30 @@ tape archive, public cloud, or your own laptop. Start a Globus app
           callbackUrl: "https://<your_jupyterhub_host>/hub/oauth_callback"
           identityProvider: "youruniversity.edu"
 
+Keycloak (OpenID Connect)
+^^^^^^^^^^^^^^^^^^^^^^^^^
 
-To add a whitelist of usernames add to the config file under `auth`:
+OpenID Connect endpoint discovery is not supported by oauthentiator,
+but you can still configure JupyterHub to authenticate with it.
```

Comment:
Could you provide a link out to documentation on how folks could learn more about this authentication option (and/or how to use it)?